ci/goreleaser.yml

```diff
@@ -0,0 +1,36 @@
+name: goreleaser^
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+permissions:
+  contents: write
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - name: Run GoReleaser
+        # Or with a condition on GoReleaser step:
+        # if: startsWith(github.ref, 'refs/tags/')
+        uses: goreleaser/goreleaser-action@v2
```

Comment:
I think they want commit SHA for actions not managed by Github.

Reply:
I checked all the other contributions, and they are alle with version tag. In the PR checklist its not written.

Reply:
done!

FYI @caarlos0 added -> 5df302e5e9e4c66310a6b6493a8865b12c555af2 from the latest release of the action!